plugin/storage/integration/elasticsearch_test.go

```diff
@@ -126,6 +125,7 @@ func (s *ESStorageIntegration) initializeES(t *testing.T, allTagsAsFields, archi
 	s.esCleanUp(t, allTagsAsFields, archive)
 	// TODO: remove this flag after ES support returning spanKind when get operations
 	s.GetOperationsMissingSpanKind = true
+	s.SkipArchiveTest = !archive
```

Comment:
I don't think this `archive` flag makes sense anymore - the archive test is always included in the harness, so it should not be conditional that ES test initializes archive reader/writer, it always needs to create both primary and archive namespaces

Reply:
Without this flag, the ES test was failing.
In the logs it was written as a nil pointer deference error.
I assumed it was coming from the SkipArchiveTrace itself, and after initialising it to !archive, the tests were successfully passing